### ISSUE TITLE
Backend API: connect 'finance' router

### ISSUE THREAD

**Issue by spencer-carver** [aymanjebril2/finanseer#14]
Reference https://github.com/aymanjebril2/finanseer/wiki/API-Contract#finance for what is needed

**Comment by spencer-carver** [comment]
NOTE: this router should accept and validate the parameters from the client, and echo a success if everything passes.

There will be a follow-up issue after #15 is completed.

**Comment by spencer-carver** [comment]
Repurposing this ticket to be the connection of front-end/back-end, as #15 added stubs for this that were intended for this ticket originally.

**Comment by spencer-carver** [comment]
Finished in #48
